SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialConditionMapFilter.h

```diff
@@ -18,7 +18,7 @@ class FSpatialConditionMapFilter
 		RepFlags.bReplay = 0;
 		RepFlags.bNetInitial = 1; // The server will only ever send one update for bNetInitial, so just let them through here.
 		RepFlags.bNetSimulated = ActorChannel->Actor->Role == ROLE_SimulatedProxy;
-		RepFlags.bNetOwner = bIsClient && ActorChannel->IsAuthoritativeClient();
+		RepFlags.bNetOwner = bIsClient;
```

Comment:
After our discussion, are we going to keep this as is?

Reply:
Looking into it, at first, I thought we may need to check the Static Component View, but maybe we don't have to.
Tested without and it seemed to pass all the tests I threw at it. Of course, still possible it's somehow wrong.

Reply:
Where are your tests at?

Reply:
The OwnerOnly test in the EngineNetTest repo should cover the worrying case here, I think. I also have a new test in a branch on the EngineNetTest with the same name (bugfix/UNR-3686...) and that also passes, but honestly that test is kind of weird, so I would not rely on it.

Reply:
*sigh*, I just realized this will break SkipOwner, since that is on the data component.